samples/customABSample/src/main/java/com/inmobi/customabsample/CustomABActivity.java

```diff
@@ -0,0 +1,89 @@
+package com.inmobi.customabsample;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.inmobi.sdk.InMobiSdk;
+import com.inmobi.sdk.SdkInitializationListener;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class CustomABActivity extends AppCompatActivity {
+
+    private static final String TAG = CustomABActivity.class.getName();
+    private Button bannerIntegration;
```

Comment:
Let's uniformly name variables. As done in Int Activity keep **m** as prefix for member variables